coredns/examples/dnsrecord-api-k-example-com_geo_weight.yaml

```diff
@@ -72,10 +72,10 @@ spec:
       recordType: A
       targets:
         - 127.0.0.1
-    - dnsName: "*.api.k.example.com"
+    - dnsName: '*.api.k.example.com'
       recordTTL: 60
       recordType: A
       targets:
         - 9.9.9.9
   providerRef:
-    name: dns-provider-credentials-coredns
+    name: dns-provider-core-dns
```

Comment:
We should try and keep these consistent with all other providers generated by local-setup "dns-provider-credentials-\<provider name\>"